### DISCUSSION TITLE
Trying to virtualize MacOS, low performance?

### DISCUSSION THREAD

**Discussion by pedrocarvalhlima** [utmapp/UTM#5740] [macOS]
Virtualizing MacOS from a mbp M2 with 16gb, tried 6 cpu + 10 gb RAM virtualization, straight from the box from UTM. Performance of streaming apps is very subpar, but youtube and social media videos do not suffer. Anyone have any thoughts on how to improve it? It seems like UTM is really not performant for video decoding, even if it's virtualizing essentially the same OS as the machine it's running on.

**Comment by DrDonk** [comment]
UTM uses Apple’s Virtualisation framework so suspect there is not much they can do if the problem is in Apples API.

**Comment by dmgoldstein1** [comment]
Hello @pseudolimonada you've gotten further than I have. I'm trying to install macOS 13 as the OS of my VM in UTM. Where do you get your IPSW images? There's a very useful website https://ipsw.me/download/VirtualMac2,1/22G120 which provides links to downloadable files at cdn-apple.com. I just want to make sure that the domain cdn-apple.com is legit. I just messaged Apple Support and they didn't know whether or not it was one of theirs, or a malicious imposter. And a WHOIS lookup just shows that it's registered by an anonymizing agent.

**Reply by dmgoldstein1** [reply]
Of course as soon as I have a post I have an idea for how to answer my own problem. I installed Little Snitch to see if that's a domain that Apple devices regularly communicate with, and I didn't even have to open the app store - one of the default rules in Little Snitch is to allow traffic to cdn-apple.com. That's good enough for me.
